### ISSUE TITLE
[Feature Request] Pantheon

### ISSUE THREAD

**Issue by ShadoweCZ** [Openarl/PathOfBuilding#543]
Pantheon is missing, that should be fixed :D

**Comment by mcd8604** [comment]
I agree. I have been manually calculating my RF degen/regen and trying to figure out how Soul of Arakaali factors in. Can this be added to PoB?

**Comment by calmchaos** [comment]
I'd also like to see Pantheon added eventually.

I'm sure it's something that's already being worked on, assuming @Openarl wants to add it at some point in time. It would be certainly a time-consuming task though, but some of the effects can have a pretty large effect.
